tests/providers/openlineage/conftest.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Copyright 2018-2023 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import logging
+import os
+from unittest.mock import patch
+
+import pytest
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def remove_redshift_conn():
+    if "REDSHIFT_CONN" in os.environ:
+        del os.environ["REDSHIFT_CONN"]
+    if "WRITE_SCHEMA" in os.environ:
+        del os.environ["WRITE_SCHEMA"]
+
+
+@pytest.fixture(scope="function")
+def we_module_env():
+    os.environ["REDSHIFT_CONN"] = "postgresql://user:password@host.io:1234/db"
+    os.environ["WRITE_SCHEMA"] = "testing"
```

Comment:
I think this will work, and it'll automatically get cleared after the test function finishes.

```suggestion
@pytest.fixture(scope="function")
def we_module_env(monkeypatch):
    monkeypatch.setenv("REDSHIFT_CONN", "postgresql://user:password@host.io:1234/db")
    monkeypatch.setenv("WRITE_SCHEMA", "testing")
```

Reply:
Removed this for now, will add when needed.